data-lineage/src/main/java/com/wipro/ats/bdre/lineage/LineageMain.java

```diff
@@ -506,50 +401,32 @@ private void handleTokTab(ASTNode nextToDestinationNode) {
 			node = (ASTNode) (nextToDestinationNode.getChild(0));
 		else
 			node  = nextToDestinationNode;
-//		String table_name = (nextToDestinationNode.getChildCount() == 1) ?
-//				getUnescapedName((ASTNode) nextToDestinationNode.getChild(0)) :
-//				getUnescapedName((ASTNode) nextToDestinationNode.getChild(0)) +
-//						"." + getUnescapedName((ASTNode) nextToDestinationNode.getChild(1));
 		String table_name = (node.getChildCount() == 1) ?
 				getUnescapedName((ASTNode) node.getChild(0)) : getUnescapedName((ASTNode) node.getChild(1));
 		outputTableList.add(table_name);
-		System.out.println("Output Table name in handleTokTab = " + table_name);
+		LOGGER.info("Output Table name in handleTokTab = " + table_name);
 		Table table = new Table(table_name, defaultHiveDbName, null, EntityType.OUTTYPE);
 		outTableNodes.add(table);
 	}
 
 	private void handleTokDir(ASTNode nextToDestinationNode, ASTNode queryNode) {
-//		System.out.println(((ASTNode) queryNode.getParent()).dump());
-//		ASTNode outputTempTableNode = (ASTNode) queryNode.getParent().getChild(1);
-//		outputTableList.add(BaseSemanticAnalyzer.getUnescapedName(outputTempTableNode));
 		for (int i=0; i<queryNode.getParent().getChildCount(); i++) {
 			ASTNode outputTempTableNode = (ASTNode)queryNode.getParent().getChild(i);
 			if (outputTempTableNode.getType() == HiveParser.TOK_TABNAME) {
 				outputTableList.add(getUnescapedName((ASTNode) outputTempTableNode.getChild(0)));
-				System.out.println("Output Table name in handleTokDir = " + getUnescapedName((ASTNode) outputTempTableNode.getChild(0)));
+				LOGGER.info("Output Table name in handleTokDir = " + getUnescapedName((ASTNode) outputTempTableNode.getChild(0)));
 				Table table = new Table(getUnescapedName((ASTNode) outputTempTableNode.getChild(0)), defaultHiveDbName, null, EntityType.OUTTYPE);
 				outTableNodes.add(table);
 
 			} else if (((ASTNode)queryNode.getParent()).getType() == HiveParser.TOK_SUBQUERY && i==1) {
 				String temp_table_name = getUnescapedName((ASTNode)queryNode.getParent().getChild(i));
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this local variable name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS00117)